src/transaction/log_global.c

```diff
@@ -143,5 +135,29 @@ log_global::get_log_prior_receiver ()
   assert (m_prior_recver != nullptr);
   return *m_prior_recver;
 }
+
+void
+log_global::update_ps_consensus_flushed_lsa ()
+{
+  assert (is_active_transaction_server ());
+
+  const log_lsa consensus_lsa = get_active_tran_server_ptr ()->compute_consensus_lsa ();
+  if (consensus_lsa == NULL_LSA)
+  {
+    return; // The number of connected node are less than the quorum
```

Comment:
This effectively means that we leave the transactions waiting on this locked? Because we're not notifying the waiting transactions if the consensus LSA cannot be calculated.
I mean we do not have to solve this now with this PR. The question is to just have a discussion on this?

The logical outcome would be to abort the waiting (and, implicitly, all future transactions)?

For the time being, I think a temporary `debug_release` somewhere is advised.

Reply:
> This effectively means that we leave the transactions waiting on this locked? 

Yes, it is. 
However, in my opinion, I don't think we should abort waiting (and future) transactions. The way I see it, it's an acceptable case that the quorum is not met temporarily and transactions wait until the quorum number is met again. It's one of the advantages of keeping going with some temporarily unavailable nodes (not only with a node lagging behind). This case would be added as a test case with the mechanism of catch-up and attaching a new node in the next epic.
 For the long term, we may need a timeout mechanism here to deal with a long period of unsatisfying quorum. For now, it's better to leave it as it is.